examples/asr/jasper_eval.py

```diff
@@ -168,18 +168,17 @@ def main():
                 )
                 beam_predictions_e1 = beam_search_with_lm(log_probs=log_probs_e1, log_probs_length=encoded_len_e1)
 
-                evaluated_tensors = neural_factory.infer(tensors=[beam_predictions_e1], use_cache=True, verbose=False,)
+                evaluated_tensors = neural_factory.infer(tensors=[beam_predictions_e1], use_cache=False, verbose=False)
```

Comment:
Any particular reason cache was set to False?

Reply:
Yes, I could not run inference after the refactoring because I run out of memory - my GPUs are only 16G and before I was using batch_size=32 but after the refactoring I couldn't run it even with a batch size of 8 - Same dataset as before. After turning cache to False, I could use batch_size of 32 again. I assume not everyone has GPUs with large memory so I assume this is a good default to have.